compiler/rustc_middle/src/ty/generic_args.rs

```diff
@@ -501,12 +501,11 @@ impl<'tcx> GenericArgs<'tcx> {
     #[inline]
     pub fn non_erasable_generics(
         &'tcx self,
-        tcx: TyCtxt<'tcx>,
-        def_id: DefId,
+        // FIXME(effects): Remove these
+        _tcx: TyCtxt<'tcx>,
```

Comment:
:thinking: why as a FIXME?

Reply:
ok, more like 13 but i'd rather not change it in this pr.